apps/financial-aid/web-osk/src/routes/status/MainPage/mainPage.tsx

```diff
@@ -87,33 +80,45 @@ const MainPage = () => {
         )}
         {loading && <LoadingDots />}
 
-        <Text as="h4" variant="h3" marginBottom={2} marginTop={[3, 3, 7]}>
-          Frekari aðgerðir í boði
-        </Text>
-        <Box marginBottom={[5, 5, 10]}>
-          <BulletList type={'ul'} space={2}>
-            <Bullet>
-              <Link
-                href={municipality?.homepage ?? ''}
-                color="blue400"
-                underline="normal"
-                underlineVisibility="always"
-              >
-                <b>Upplýsingar um fjárhagsaðstoð</b>
-              </Link>
-            </Bullet>
-            <Bullet>
-              <Link
-                href={`mailto: ${municipality?.email}`}
-                color="blue400"
-                underline="normal"
-                underlineVisibility="always"
-              >
-                <b> Hafa samband</b>
-              </Link>
-            </Bullet>
-          </BulletList>
-        </Box>
+        {(municipality?.homepage || municipality?.email) && (
```

Comment:
Wondering if we should draw this out as a component and take these variables as optional props?